### PR TITLE
Contact form pull request

### PR DESCRIPTION
**Pull request by ntimanu**
In this project we:

        - Added contact form
        
        - Online submission
        
        - Added buttons design

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -464,124 +422,119 @@ ul li {
...
+#form-id {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 52px;
+  gap: 1rem;
 }
 
+.inputtxt1,
 .inputtxt2 {
```
Comment (usorfaitheloho) [inline style.css:549]: Good job implementing some of the project requirements, however, as stated in the requirement, you are to stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding).Therefore, i'd kindly make the following suggestions 👯  :

- [x] In your design, you used the wrong color for your all your `placeholder` text, which is different from the figma design given. please ensure you use the approraite colour to give you a good aesthetic that'd attract hiring managers 😄
 
- [x] Could you kindly increase the padding for your `input text` as it looks really close to the input border in contrast to what we have in the design, please take a close look at it and implement the changes

- [x] Please implement the `border-radius` property for the `textarea` as what you have looks different from what we have in the design and might not be too visually appealing 😄 

- [x] kindly use the right button interaction styling for hover `font color`, which should be `#ffff`, and the right background color the `pressed` button interaction. Please use the right color styling for the `button interaction`   

Button interaction :

image

Your design:

image

Figma design:

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
## Hi @ntimanu and @kkmanuu   :clap:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## To Highlight :clap:  
:arrow_forward: Git/Github flow best practices has been followed :heavy_check_mark: 
:arrow_forward: Good code quality :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (ntimanu) [reply style.css:549]: Done

File: style.css [hunk 2]
```diff
@@ -464,124 +433,120 @@ ul li {
...
```
Comment (divinecharlotte) [inline style.css:549]: - [x] kindly as requested by the previous reviewer consider checking on this 👉  In your design, you used the wrong color for your all your placeholder text, which is different from the Figma design given. please ensure you use the appropriate colour to give you a good aesthetic that'd attract hiring managers 😄

File: style.css [hunk 3]
```diff
@@ -593,29 +558,15 @@ ul li {
...
-  position: relative;
-  left: 24px;
-  right: 0;
-  top: 400px;
-  bottom: 0;
-  height: 170px;
-  width: 327px;
   background: #fff;
   border: 1px solid #cfd8dc;
+  border-radius: 8px;
```
Comment (divinecharlotte) [inline style.css]: - [x] Kindly consider increasing the padding for your input text as it looks really close to the input border in contrast to what we have in the design, please take a close look at it below and implement the changes and also consider increasing the width of the get in touch  button  as well to make an exact duplicate of the provided design 
- your design 👇 
Screen Shot 2023-05-11 at 13 45 26

- Figma design 👇 
Screen Shot 2023-05-11 at 13 48 31

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @ntimanu       , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- Clean code 👍
- linters are passing 👍 




 

## Required Changes ♻️

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @ntimanu       , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- Clean code 👍
- linters are passing 👍 




 

## Required Changes ♻️

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (ntimanu) [reply style.css:549]: Done

File: style.css (hunk 3 above)
Comment (ntimanu) [reply style.css]: Done

File: style.css [hunk 4]
```diff
@@ -361,9 +309,18 @@ ul li {
...
+}
+
+.contactme button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  gap: 10px;
```
Comment (levy002) [inline style.css]: - [ ] Kindly make sure as requested by the previous reviewer to align the `Get in touch` text correctly as the button looks in the provided Figma design. Adjust your width and padding in order to match the design.


    - ### YOUR DESIGN
    
    image

    - ### FIGMA DESIGN
    
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @ntimanu   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @ntimanu 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**